.github/ISSUE_TEMPLATE/new-member-onboarding.md

```diff
@@ -0,0 +1,18 @@
+---
+name: New member onboarding
+about: Checklist of actions to take when someone joins the group
+title: New member onboarding for [Name]
+labels: admin
+assignees: dryan
+
+---
+
+- [ ] Request they be added to the https://github.com/django org if they aren't already (email ops@djangoproject.com to get added)
+- [ ] Add member to https://github.com/orgs/django/teams/coc-committee/members
+- [ ] Add email account to: https://groups.google.com/g/django-coc-committee
+- [ ] Add them to the `#ask-coc` and `#coc-working-group` channels in the DSF Slack
+
+If the person is serving as Chair, Co-chair, or Board Liaison also do the following:
+
+- [ ] Make them a maintainer in https://github.com/orgs/django/teams/coc-committee/members
+- [ ] Make them a manager in https://groups.google.com/g/django-coc-committee
```

Comment:
We could also have them set up as "Channel Manager" in Slack? Which is the role needed to add/remove people to the channel IIRC. Both for the group channel and ask-coc